src/Dfe.PlanTech.Infrastructure.Contentful/Helpers/ContentfulSetup.cs

```diff
@@ -41,8 +41,7 @@ public static IServiceCollection SetupContentfulClient(this IServiceCollection s
 
 
         services.SetupRichTextRenderer();
-
-        setupClient(services.AddHttpClient<ContentfulClient>());
+        setupClient(services.AddHttpClient<ContentfulClient>("contentful"));
```

Comment:
reminder to revert

Reply:
[73c9a30](https://github.com/DFE-Digital/sts-plan-technology-for-your-school/pull/900/commits/73c9a30660fb5862794c88307b9c9b84c997f1ea) Thanks